Header header
int32[] category
geometry_msgs/Point[] location
